### PR TITLE
Adding support for Fedora 28 by adding fedora-gpg-keys

### PR DESCRIPTION
**Pull request by edgan**

### CODE REVIEW

**Comment by stgraber** [comment]
Please include the required `Signed-off-by: Firstname Lastname <email>` line in your commit message

File: templates/lxc-fedora.in [hunk 1]
```diff
@@ -812,6 +812,13 @@ setup_repositories()
     local release="${3}"
     local mirror="${4}"
 
+    PKGS=(fedora-release-${release} fedora-repos-${release}
```
Comment (LicoMonch) [inline templates/lxc-fedora.in]: the opening bracket '(' breaks the whole script ;)

Comment (brauner) [reply templates/lxc-fedora.in]: We need a signed-off by before we can even look at the code. :(

Comment (LicoMonch) [reply templates/lxc-fedora.in]: well, at least for me it doesn't matter, as the script (even with the line fixed with quotes) still isn't able to get me a F28 container on a debian host ;)

**Comment by perlun** [comment]
(This one can be closed, superseded by #8)

**Pull request closed without merging**